### DISCUSSION TITLE
Urgent Assistance Needed: Domain Transfer Out Failure

### DISCUSSION THREAD

**Discussion by dommi10** [vercel/community#7440] [Help] [answered]
Dear Vercel Support Team,

I hope this message finds you well. I am writing to seek urgent assistance regarding the transfer out of my domain, which I initiated five days ago before its expiration.

Domain Details:

	•	Domain Name: abecha-rdc.com
	•	Initiation Date: July 7th,2024

Despite following the necessary steps to transfer my domain to another registrar, the process has not been completed successfully. Given that the transfer was initiated prior to the domain’s expiration, I am concerned about the status and would appreciate your prompt assistance to resolve this matter.

Issues Encountered:

	•	The transfer out process seems to have stalled.
	•	I have not received any further updates or instructions regarding the transfer status.
	•	The expiration date is approaching, adding urgency to this matter.

Could you please investigate the issue and provide guidance on the necessary steps to ensure the successful transfer of my domain? Any help you can provide would be greatly appreciated.

Thank you for your attention and prompt response.

Best regards,
Dom’s Buhendwa
domsbuhendwa2@gmail.com

**Comment by pawlean** [comment]
Hi @dommi10!

Can you give us more information on the error you're seeing? This would help us understand how to help and debug.

> my domain to another registrar

Are you moving your domain you purchased from us to another registrar?

**Reply by dommi10** [reply]
![image](https://github.com/user-attachments/assets/12b55716-0f16-4d8a-a17b-a4ce00b651f3)


As you in image, the notice me that domain is failed 
And yes I move my domain to another register (Cloudflare)

**Reply by pawlean** [reply]
It seems that this is a question you'll need to ask Cloudflare for more information.

**Reply by dommi10** [reply]
Cloudflare told me that vercel block the transfer to be completed

**Reply by pawlean** [reply]
Let me share this internally!

**Reply by dommi10** [reply]
> Let me share this internally! 

Thanks you 🙌

**Reply by sebastianroming** [reply, marked as answer]
Hi @dommi10, 
For domain transfers with certain TLDs, there's typically a 5 days grace period during which the transfer process can still be cancelled. After these 5 days, the transfer will automatically be approved. We've now manually approved the transfer process and the domain should be with your new registrar now. Our apologies for any inconveniences this may have caused.

**Reply by dommi10** [reply]
> Hi @dommi10, 
> 
> For domain transfers with certain TLDs, there's typically a 5 days grace period during which the transfer process can still be cancelled. After these 5 days, the transfer will automatically be approved. We've now manually approved the transfer process and the domain should be with your new registrar now. Our apologies for any inconveniences this may have caused.

Thanks a lot 🫡

[a comment by an automated account is not shown]
